### PR TITLE
ZIFA: Dimensionality reduction for zero-inflated single-cell gene

### PR DESCRIPTION
**Pull request by Yue-Jiang**
*  [x] I have read the guidelines above.
* [x] This PR adds a new recipe. 
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

expression analysis

### CODE REVIEW

**Comment by Yue-Jiang** [comment]
Noticed that this package is only compatible with python2, so please don't accept this PR for now. Sorry about the confusion.

File: recipes/zifa/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+package:
+  name: zifa
+  version: "0.1.0"
+
+source:
+  git_rev: 
+  git_url: https://github.com/epierson9/ZIFA
+
```
Comment (bgruening) [inline recipes/zifa/meta.yaml]: You can add this:

```
build:
  number: 0
  skip: True # [not py27]
```

File: recipes/zifa/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,24 @@
+package:
+  name: zifa
+  version: "0.1.0"
+
+source:
+  git_rev:
```
Comment (bgruening) [inline recipes/zifa/meta.yaml]: Please refer to some revision: e.g. 627f7794d455c892407c3c57d31189aef0154875

**Comment by Yue-Jiang** [comment]
Thanks @bgruening, i've made the changes.

**Comment by bgruening** [comment]
@Yue-Jiang looks good to me. Feel free to merge :)

**Pull request merged**